src/web/views/Components/Search/Search.vue

```diff
@@ -113,6 +119,27 @@ export default {
           "Use this boolean to add an autofocus on the input search",
           "",
         ],
+        [
+          "radius",
+          "Boolean",
+          "false",
+          "Use this prop to use radius search bar",
+          "",
+        ],
+        [
+          "square",
+          "Boolean",
+          "false",
+          "Use this prop to use radius search bar",
```

Comment:
Small remark : I guess it's "square" here, instead of "radius"

Reply:
Yes, you're right. :)
I'll fix this.